pkg/kv-cache/indexer.go

```diff
@@ -54,7 +54,7 @@ func NewDefaultConfig() *Config {
 type Indexer struct {
 	tokensIndexer   prefixstore.Indexer // gets tokens for a prompt
 	tokensProcessor TokenProcessor      // turns tokens to kv block keys
-	kvBlockIndexer  *kvblock.Indexer    // looks up pods for block keys
+	kvBlockIndex    kvblock.Index       // looks up pods for block keys
```

Comment:
Q: the move from an active "Indexer" to a more passive "Index" is intended?

Reply:
The timeline behind this change was that a `kvblock.Indexer` was still present when I added the concept of `Index(Backend)`, then concluded that the first should be dropped. That made bias for this name. If through your eyes it is less sensible then a change is fine by me.